src/rosetta-helpers.ts

```diff
@@ -1012,7 +1016,11 @@ export async function getValidatedFtMetadata(
   const tokenContractId = assetIdentifier.split('::')[0];
   const ftMetadata = await db.getFtMetadata(tokenContractId);
   if (!ftMetadata.found) {
-    throw new Error(`FT metadata not found for token: ${assetIdentifier}`);
+    if (tokenMetadataMode() === TokenMetadataMode.warning) {
+      logger.warn(`FT metadata not found for token: ${assetIdentifier}`);
+    } else {
+      throw new Error(`FT metadata not found for token: ${assetIdentifier}`);
```

Comment:
This is fine for the hotfix, but I think in a future PR we should check if a given FT's ABI matches the SIP-09 ABI before throwing. 

It actually might need to go further and check 1) does the token match SIP-09 ABI, and 2) is the metadata missing because of a recoverable error e.g. network timeouts vs a Clarity contract error. Otherwise a single SIP-09 compliant looking FT could still have some contract error and completely block the strict mode from working.

Reply:
Good point. I'll add a TODO comment to that effect